### PR TITLE
bpo-20582: add link for manpage for flags on getnameinfo()

### PR DESCRIPTION
**Pull request by eamanu**

### CODE REVIEW

**Comment by matrixise** [comment]
@JulienPalard do we need a blurb entry for this issue? or add the `skip-news` label.

LGTM

File: Doc/library/socket.rst [hunk 1]
```diff
@@ -752,6 +752,9 @@ The :mod:`socket` module also offers various network-related services:
    For IPv6 addresses, ``%scope`` is appended to the host part if *sockaddr*
    contains meaningful *scopeid*. Usually this happens for multicast addresses.
 
+   For more information about *flags* you can consult the
+   `manpage <http://man7.org/linux/man-pages/man3/getnameinfo.3.html>`_.
```
Comment (JulienPalard) [inline Doc/library/socket.rst]: You could add the name of the manpage so people prefering to consult it locally can easily do.
```suggestion
   `getnameinfo manpage <http://man7.org/linux/man-pages/man3/getnameinfo.3.html>`_.
```

**Comment by eamanu** [comment]
Ready @JulienPalard

**Comment by csabella** [comment]
Please take a look at Serhiy's comment on the bug tracker.

**Comment by eamanu** [comment]
oops!

**Comment by eamanu** [comment]
Sorry everybody for the noise. I was wrong on the push. 

Thanks @csabella  I make the suggestion

**Comment by eamanu** [comment]
ping @JulienPalard :-)

File: Doc/library/socket.rst [hunk 2]
```diff
@@ -752,6 +752,8 @@ The :mod:`socket` module also offers various network-related services:
    For IPv6 addresses, ``%scope`` is appended to the host part if *sockaddr*
    contains meaningful *scopeid*. Usually this happens for multicast addresses.
 
+   For more information about *flags* you can consult on :manpage:`getnameinfo(3)`.
```
Comment (brettcannon) [inline Doc/library/socket.rst]:
```suggestion
   For more information about *flags* you can consult :manpage:`getnameinfo(3)`.
```

**Review by brettcannon — CHANGES_REQUESTED** [review]

**Comment by eamanu** [comment]
I have made the requested changes; please review again

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**